api/rest/v1/bulk_observations_point_linked.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: default
+title: Single Observation (linked)
+nav_order: 9
+parent: REST (v1)
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/point/linked
+---
+
+# /v1/bulk/observations/point/linked
+
+Retrieve single [observations](/glossary.html#observation) of multiple [variables](/glossary.html#variable) at a set date for entities linked to a parent entity by the same property.
+
+This is useful for retrieving observations for all places within a parent place. For example, this could be getting the population of women in 2018 for all states in the United States.
```

Comment:
s/parent/ancestor, and maybe link the "linked" concept graph here?

Reply:
Same for other places to change parent to ancestor

Reply:
Done! Also s/parent/ancestor in bulk/observations/series/linked